app/views/cookies/_form.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_for @cookie, local: true, url: '/cookies/settings', html: {class: 'app-js-only'} do |f|
```

Comment:
- Remember to use the relevant rails route instead of the static url `/cookies/settings`
- Is the class `app-js-only` still required?

Reply:
I've changed the form to use the Rails path. The class is still required, we've implemented a slightly different banner and form for where JS is not enabled, so the form will not be rendered if the user has disabled JS

Reply:
relevant rails route used now